server/paths/manage.ts

```diff
@@ -5,6 +5,7 @@ const singlePremisesPath = premisesPath.path(':premisesId')
 
 const bookingsPath = singlePremisesPath.path('bookings')
 const bookingPath = bookingsPath.path(':bookingId')
+const newBookingPath = singlePremisesPath.path('bed/:bedId/bookings/new')
```

Comment:
Should probably be `/beds` here. Arguably we should nest this under rooms too, but that's probably overkill